Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -365,39 +372,148 @@ def start_chrome_headless(chrome_port, chrome_binary=CHROME_EXE, user_options=""
     return None, None
 
 
-def terminate_chrome(browser):
-    global CHROME_PROCESS
-    demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
+def terminate_chrome(chrome_port=None, killall=False):
+    processes = subprocess.check_output(['ps', 'auxww'], stderr=subprocess.STDOUT, text=True).splitlines()
+    chrome_renderer_identifiers = ["--type=renderer"]
+    chrome_identifiers = ["chrom", "headless"] if killall else ["chrom", "headless", f"--remote-debugging-port={chrome_port}"]
+    process_in_list = [process for process in processes
+                       if all(identifier in process for identifier in chrome_identifiers)
+                       and not any(identifier in process for identifier in chrome_renderer_identifiers)]
 
-    threading.excepthook = excepthook_recv_loop
+    if killall:
+        pids = [int(process.split()[1]) for process in process_in_list]
+    else:
+        process_string_representation = process_in_list[0]
+        pids = [int(process_string_representation.split()[1])]
 
-    if CHROME_PROCESS:
-        demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
-        CHROME_PROCESS.kill()
-        CHROME_PROCESS = None
+    for pid in pids:
+        process = psutil.Process(pid)
+        if process:
+            demisto.debug(f'terminate_chrome, {process=} on port {chrome_port}')
+            process.kill()
 
     demisto.debug('terminate_chrome, Finish')
 
 
-def ensure_chrome_running():  # pragma: no cover
-
-    # Check if we have a file with the port.
-    # If we have a file - Try to use it.
-    # If there's no file, or we cannot use it - Find a free port
-    browser = None
-    chrome_port = read_info_file(PORT_FILE_PATH)
-    if chrome_port:
-        browser = is_chrome_running_locally(chrome_port)
-        if browser:
-            return browser, chrome_port
+def chrome_manager():
+    instance_id = demisto.callingContext.get('context', {}).get('IntegrationInstanceID')
+    chrome_options = demisto.callingContext.get('params', {}).get('chrome_options')
+    demisto.log(f"in chrome_manager {chrome_options=}, {instance_id=}")  # TODO: Remove log
+    demisto.log(f"{instance_id=}")  # TODO: Remove log
+
+    if not instance_id:
+        instance_id = 'NO-INSTANCE-ID'
+    if not chrome_options:
+        chrome_options = 'None'
+
+    info = read_info_file(CHROME_INSTANCES_FILE_PATH)
+    demisto.log(f"{info=}")  # TODO: Remove log
+
+    instance_id_to_chromes_options, chrome_options_to_port, instance_id_to_port = get_chrome_instances_info_dictionaries(info)
+
+    if info:
+        demisto.log("there is an info")  # TODO: Remove log
+        info_without_new_lines = info.strip()
+        splitted_info = info_without_new_lines.splitlines()
```

Comment:
You don't need `info_without_new_lines`.
```suggestion
        splitted_info = info.strip().splitlines()
```